books.rb

```diff
@@ -0,0 +1,27 @@
+class Book
+  attr_reader :id
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @id = Random.rand(1..1000)
+    @title = title
+    @author = author
+  end
+end
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] It would be better if you add the class Rental to its own file `rental.rb` with this you will have a more organized project. 😄 👍